manifests/bosh-manifest/deployments/aws/010-bosh-aws-cpi.yml

```diff
@@ -1,3 +1,6 @@
+# When updating the version of the CPI here, the version used in the
+# bosh-init container must also be updated so that the cached CPI compile
+# will be used.
```

Comment:
Could you move this comment just above or below `name: bosh-aws-cpi` (like `concourse-base.yml`) so that it stays relevant if we happen to add any more releases to this manifest.